yt_dlp/extractor/rtvslo.py

```diff
@@ -164,3 +175,28 @@ def _real_extract(self, url):
             'series': meta.get('showName'),
             'series_id': meta.get('showId'),
         }
+
+    def _get_show_urls(self, url, v_id):
+        html = self._download_webpage(url, v_id)
+        all_urls = set(re.findall(r'<a.*href=\"(/arhiv/.+)\".*aria-label=.*>', html))
+        parsed = urlparse(url)
+        res = re.findall(r'<title>(.*)</title>', html)
+        title = res[0] if len(res) != 0 else None
+        return title, [f'{parsed.scheme}://{parsed.netloc}{url}' for url in all_urls]
+
+    def _real_extract(self, url):
+        url_id = self._match_id(url)
+        if 'oddaja' in url:
+            # supplied URL is a shows homepage / like a channel or playlist
+            title, urls = self._get_show_urls(url, url_id)
+            entries = [self._get_video_info(self._match_id(url)) for url in urls]
+            entries.sort(key=lambda x: x['id'])
+            return {
+                '_type': 'playlist',
+                'id': url_id,
+                'title': title,
+                'entries': entries,
+                'playlist_count': len(urls)
+            }
```

Comment:
Don't extract the info for all videos ahead of time
```suggestion
            return self.playlist_result(
                [self.url_result(url, RTVSLOIE, url_id) for url in urls], url_id, title)
```